cfme/storage/volume.py

```diff
@@ -255,6 +267,17 @@ def create_backup(self, name, incremental=None, force=None):
 
         wait_for(lambda: self.backups > 0, delay=20, timeout=1000, fail_func=self.refresh)
 
+    def create_snapshot(self, name):
+        """create snapshot of cloud volume"""
+        view = navigate_to(self, 'Snapshot')
+        view.snapshot_name.fill(name)
+
+        view.save.click()
+        view.flash.assert_success_message('Snapshot for Cloud Volume "{}" '
+                                          'created'.format(self.name))
+
+        wait_for(lambda: self.snapshots > 0, delay=20, timeout=1000, fail_func=self.refresh)
```

Comment:
**REQUIRED**: This should probably check the snapshot count first and then check against the previous snapshot count.

Reply:
my bad; I thought we are creatiing new volume for test so snapshot cout always zero; but good to compair wth initial snapshot count for famework flexibility.